meta_test.go

```diff
@@ -56,14 +56,55 @@ func (s *MetaSuite) TestReadMetaVersion2(c *gc.C) {
 	c.Assert(meta.Terms, gc.HasLen, 0)
 }
 
+func (s *MetaSuite) TestValidTermFormat(c *gc.C) {
+	valid := []string{
+		"foobar",
+		"foobar/27",
+		"foo/003",
+		"owner/Foo-bar",
+		"owner/foobar/27",
+		"owner/foobar",
+		"owner/foo-bar",
+		"owner/foo_bar",
+		"own-er/foobar",
+		"own_er/foobar",
+		"ibm/j9-jvm/2",
+		"term_123-23aAf/1",
+		"@les/term/1",
```

Comment:
I don't agree with this being a valid owner if it would not be allowed in a charm URL.